src/components/Form/Form.md

```diff
@@ -3,6 +3,12 @@ import validateTruthy from '../../validators/validateTruthy'
 import validateMinMaxFactory from '../../validators/validateMinMax'
 import { TitleLarge, TextInput, Spacer, Button, InfoMessage, ZipInput, ZipInputValidator } from '../index'
 let count = 0
+
+function validateCustom(x) {
```

Comment:
This will allow consumers / developers to see more clearly how these kick in or not